MultiTable/src/Utils.js

```diff
@@ -28,7 +28,11 @@ module.exports = class Utils {
       subjects: {},
     };
     if (subject.absolutePath) {
-      inv.subjects[subject.absolutePath.toLowerCase()] = subject;
+
+      // need deep copy to avoid modifying the input
+      const _subj = JSON.parse(JSON.stringify(subject));
```

Comment:
there may be performance implications to this. An alternative is to make another object and attach all the fields that are not `children`

Reply:
Yeah I think parse/stringify could be expensive here. Can you perf test this hypothesis?